Code to add trailing / to DESTDIR is broken, remove it as the path always
has the slash anyway.

diff -rN -u old-wxcore-0.11.1.2/makefile.lib new-wxcore-0.11.1.2/makefile.lib
--- old-wxcore-0.11.1.2/makefile.lib	2009-08-07 13:54:39.000000000 +0200
+++ new-wxcore-0.11.1.2/makefile.lib	2009-08-07 13:54:39.000000000 +0200
@@ -41,15 +41,6 @@
 #--------------------------------------------------------------------------
 
 #--------------------------------------------------------------------------
-# Support DESTDIR
-#--------------------------------------------------------------------------
-
-# append a trailing "/" if DESTDIR is not empty
-ifneq ($(DESTDIR),)
-DESTDIR=$(patsubst %/,%,$(DESTDIR))/
-endif
-
-#--------------------------------------------------------------------------
 # Functions  on files and directories
 #--------------------------------------------------------------------------
 
